alpharaw/match/psm_match.py

```diff
@@ -600,6 +663,9 @@ def get_best_matched_intens(
     frag_start_idxes: np.ndarray,
     frag_stop_idxes: np.ndarray,
 ):
+    """
+    TODO Deprecated
```

Comment:
should we raise a deprecationwarning here? (also below)

Reply:
Well, it is numba function...